examples/fabric-admin/device_manager/DeviceManager.cpp

```diff
@@ -132,11 +131,15 @@ void DeviceManager::OpenRemoteDeviceCommissioningWindow(EndpointId remoteEndpoin
     // Open the commissioning window of a device from another fabric via its fabric bridge.
     // This method constructs and sends a command to open the commissioning window for a device
     // that is part of a different fabric, accessed through a fabric bridge.
-    StringBuilder<kMaxCommandSize> commandBuilder;
+    StringBuilder<512> commandBuilder;
+
+    // Use random discriminator to have less chance of collission.
+    uint16_t discriminator = Crypto::GetRandU16() % 4096; // 4096 to include the upper limit 4095
```

Comment:
Rather than using the "4096" value directly, it would be more appropriate to define the value 4096 as a const value and use that variable.